libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -126,6 +127,13 @@ const AP_Param::GroupInfo AP_BLHeli::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BDMASK",  11, AP_BLHeli, channel_bidir_dshot_mask, 0),
 #endif
+    // @Param: REMASK
```

Comment:
this will conflict with loading old params
maybe SERVO_BLH_RVMASK?

Reply:
Fixed